dv/uvm/riscv_dv_extension/testlist.yaml

```diff
@@ -102,7 +102,27 @@
     +no_ebreak=0
   rtl_test: core_ibex_base_test
 
-- test: riscv_ebreak_debug_mode_test
+- test: riscv_debug_basic_test
+  description: >
+    Randomly assert debug_req_i, random instruction sequence in debug_rom section
+  iterations: 10
+  gen_test: riscv_instr_base_test
+  gen_opts: >
+    +no_ebreak=1
+    +instr_cnt=6000
+    +no_branch_jump=1
+    +no_csr_instr=1
+    +no_fence=1
+    +num_of_sub_program=0
+  rtl_test: core_ibex_base_test
+  sim_opts: >
+    +enable_debug_seq=1
+  compare_opts: >
+    compare_final_value_only=1
+    verbose=1
+
+# TODO(udij) - this test is later in testlist - ignore for now
```

Comment:
If it's ignored, should the iteration be set to 0 for now?

Reply:
Sure, good point.